src/format.c

```diff
@@ -343,6 +360,29 @@ int32_t generate_csp_header() {
   memcpy(header + offset, &csp_time, sizeof(double));
   offset += mdsize;
 
+  /// chunk: sizes with border
+  /// the dimensions with border cells added
+  /// follows by flag value that is 1 if written with borders
+  /// and 0 if written without
+  int32_t csp_borders_flag_32 = (int32_t)csp_borders_flag;
+  mdsize = 4 * sizeof(int);
+  cksize = 8 + mdsize;
+  memcpy(header + offset, &cksize, 4);
+  offset += 4;
+  memcpy(header + offset, "BORD", 4);
+  offset += 4;
+  memcpy(header + offset, &H, sizeof(int));
```

Comment:
`int` does not have a fixed size. Prefer `int32_t` or similar for working with bytes.

Reply:
I was trying to be consistent with the rest of the function, and I think whoever wrote it was trying to make it flexible so it could deal with varying size integers and big and little endian. However, it looks like the function is not internally consistent and mixes int32_t with int, so I'll just make it all int32_t.

Reply:
Probably most of it should all be `uint32_t` since bitwise operations are not (I think) well-defined on signed integers. But if it works, maybe we should stop mucking with it.